helpers/timesHelper.js

```diff
@@ -0,0 +1,167 @@
+const moment = require('moment-timezone');
+const logger = require('./logger');
+
+const DEFAULT_TIMEZONE = 'Europe/Berlin';
+
+let schoolTimezone;
+let userTimezone;
+let sameTimezone = true;
+
+/**
+ * @return {String} UTC offset as string based on current timezone, e.g +01:00
+ */
+const getUtcOffset = () => moment().format('Z');
+
+const getUserTimezone = (req) => (((req || {}).cookies || {}).USER_TIMZONE);
+
+/**
+ * @param {String} res result object containing shool data information
+ * sets default timezone if school timezone differs from the user timezone
+ */
+const setDefaultTimezone = (req, res) => {
+	schoolTimezone = (res.locals.currentSchoolData || {}).timezone;
+	userTimezone = getUserTimezone(req) || res.locals.currentTimezone;
+
+	if (schoolTimezone) {
+		res.locals.currentTimezone = schoolTimezone;
+	} else {
+		res.locals.currentTimezone = DEFAULT_TIMEZONE;
+	}
+	moment.tz.setDefault(res.locals.currentTimezone);
+	sameTimezone = res.locals.currentTimezone === userTimezone;
+
+	res.locals.currentTimezoneOffset = getUtcOffset();
+	res.locals.userTimezone = userTimezone;
+	res.locals.sameTimezone = sameTimezone;
+
+	logger.info(`timesHelper: instance timezone "${res.locals.currentTimezone} (${res.locals.currentTimezoneOffset})"`);
```

Comment:
please, use logger.debug instead: https://docs.hpi-schul-cloud.org/display/TSC/Error+Handling+and+Logging#ErrorHandlingandLogging-LoglevelsandLogsources